tests/settings/ConfigurationGatheringTest.py

```diff
@@ -63,13 +63,13 @@ def test_gather_configuration(self):
 
         self.assertEqual(len(local_bears['default']), 0)
 
-    def test_default_coafile_parsing(self):
+    def test_base_coafile_parsing(self):
```

Comment:
Create a file `default_coafile` in `tests/settings/section_manager_test_files` and for its contents refer the `base_coafile` in that directory. and design your assert equal statement accordingly. Contents of `base_coafile` and `default_coafile` must be similar but not exactly the same. Also check if the warning statement is shown.